coalib/bearlib/aspects/collections.py

```diff
@@ -1,15 +1,23 @@
 from coalib.bearlib.aspects.meta import issubaspect, assert_aspect
+from inspect import isclass
 
 
 class aspectlist(list):
     """
     List-derived container to hold aspects.
     """
 
-    def __init__(self, seq=()):
+    def __init__(self, seq=(), bear=None):
         super().__init__(map(assert_aspect, seq))
+        self.bear = bear
 
     def __contains__(self, aspect):
+        if self.bear is not None and not isclass(aspect):
+            for language in self.bear.languages:
+                if type(aspect.language) is type(language):
+                    break
```

Comment:
This can be changed to
`if aspect.language not in self.bear.languages:
     return False`
after https://github.com/coala/coala/pull/4109/ is merged